### ISSUE TITLE
Heroku Deployment Logs Failure even after migrating db to Postgres

### ISSUE THREAD

**Issue by msyleung** [msyleung/tandemApprenticeship2019#1]
```
2019-12-14T23:44:13.587549+00:00 app[web.1]: bundler: failed to load command: rackup (/app/vendor/bundle/ruby/2.4.0/bin/rackup)
NameError: uninitialized constant ApplicationController on Heroku
2019-12-14T23:44:13.587614+00:00 app[web.1]: /app/app/controllers/plants_controller.rb:1:in `<top (required)>'
...
2019-12-14T23:44:13.640791+00:00 heroku[web.1]: State changed from starting to crashed
2019-12-14T23:44:13.629673+00:00 heroku[web.1]: Process exited with status 1
2019-12-14T23:44:19.412397+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=tandem-apprenticeship2019.herokuapp.com request_id=0ec397e1-52c5-40a6-9dfe-8f10a606e424 fwd="75.18.102.27" dyno= connect= service= status=503 bytes= protocol=https
```

It worked on my machine :-( 
but not on Heroku.

**Comment by msyleung** [comment]
Needed `require './config/environment'` on my controllers. 
Why doesn't it load & require with `config.ru` on heroku?

**Comment by msyleung** [comment]
It's still a randomly occurring issue so the "fix" above did not solve it.
Neither did these (Procfile, heroku restart): https://dev.to/lawrence_eagles/causes-of-heroku-h10-app-crashed-error-and-how-to-solve-them-3jnl 

There is probably a missing require somewhere, but I'm not sure 😥

**Comment by msyleung** [comment]
Resolved with commit: [44efb4c9d583882b80454aee3c8e00bb602c8dcb](https://github.com/msyleung/tandemApprenticeship2019/commit/44efb4c9d583882b80454aee3c8e00bb602c8dcb)
Spacing matters 🙏
